### PR TITLE
Update theme values and add typography components

### PR DESCRIPTION
**Pull request by martinmcneela**
Ticket: no

Why this is needed: Update theme values and add typography components

Link to Figma doc: https://www.figma.com/file/OpCqZzeuBbb8NBshcsBRpc/Edozo-maps-DS?node-id=0%3A1

### CODE REVIEW

File: src/DisplayBox/DisplayBox.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { StyledDisplayBox, StyledDisplayBoxColor, StyledDisplayBoxWrapper, StyleProps } from './DisplayBox.styles';
+
+export const DisplayBox: React.FC = (props: StyleProps) => (
+  <StyledDisplayBox {...props}>
+    <StyledDisplayBoxColor bg={props.bg} size={props.size} radius={props.radius} />
```
Comment (zac-robinson) [inline src/DisplayBox/DisplayBox.tsx]: I know its meant to be short lived, but can bg change to background

**Comment by martinmcneela** [comment]
https://edozo.github.io/mechanical-wombat/

File: src/Theme/Space.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { DisplayBoxTitle, DisplayBoxSubTitle, DisplayBoxBody } from '../DisplayBox/DisplayBox.styles.ts';
+
+export const spaces = [
+  { size: '4px', name: 'xxsmall' },
+  { size: '8px', name: 'xsmall' },
+  { size: '16px', name: 'small' },
+  { size: '24px', name: 'base' },
+  { size: '32px', name: 'largel' },
+  { size: '48px', name: 'xlarge' },
+  { size: '64px', name: 'xxlargel' },
```
Comment (zac-robinson) [inline src/Theme/Space.stories.mdx]: typo

File: src/Theme/Space.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<Meta title="Theme|Space" />
+
+# Spacing
+
+This is the spacing that will be used throught all edozo apps (customer facing). The design files can be found [here](https://www.figma.com/file/OpCqZzeuBbb8NBshcsBRpc/Edozo-maps-DS?node-id=155%3A1235).
+
+<Story name="Default">
+  <DisplayBoxWrapper>
+    {spaces.map(space => (
+      <DisplayBox size={space.size} radius="xsmall">
```
Comment (zac-robinson) [inline src/Theme/Space.stories.mdx]: Can you change radius to something more like borderRadius so it is less confusing

**Comment by zac-robinson** [comment]
This looks good, would like if further down the road we can make it pull out the colour hex values from the theme so that these stories update automatically, but happy for how it is now

File: src/styled.d.ts [hunk 4]
```diff
@@ -64,16 +64,39 @@ declare module 'styled-components' {
...
+        h1: string;
+        h2: string;
+        h3: string;
+        h4: string;
+        h5: string;
+        h6: string;
+        body: string;
         bodySmall: string;
         label: string;
+        small: string;
```
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/styled.d.ts]: agree, I think we need to align with Silvia when she gets back. Won't do it in this PR.

File: src/styled.d.ts [hunk 5]
```diff
@@ -64,16 +64,39 @@ declare module 'styled-components' {
...
```
[the remark this answers is not in the document]
Comment (martinmcneela) [reply src/styled.d.ts]: as above

**Comment by martinmcneela** [comment]
> This looks good, would like if further down the road we can make it pull out the colour hex values from the theme so that these stories update automatically, but happy for how it is now

100% agree, the more automated we can make it the better.

**Pull request merged**